seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -36,25 +36,21 @@
-        "<code>console.log(camper); </code>",
-        "<code>var camper = 'David'; </code>",
-        "<code>// logs undefined</code>",
-        "The code runs in the following order: the variable camper is declared as undefined, camper is logged, and then David is assigned to camper. This code will run without an error.",
-        "A new keyword was introduced in ES6 to solve the problems with the var keyword, called let. With the let keyword, all the examples we just saw will cause an error to appear. We can no longer overwrite variables or use a variable before we declare it. Some modern browsers requires you to add \"use strict\" to the top of your code before you can able to use the new features in ES6",
-        "Let's try using the let keyword;",
-        "Instructions",
-        "Fix the code so it only uses the let keyword and make the errors go away.",
-        "Note",
-        "Remember to add \"use strict\" to the top of your code."
+        "One of the biggest problems with declaring variables with the <code>var</code> keyword is that you can overwrite variable declarations without an error.",
+        "<blockquote>var camper = 'James';<br>var camper = 'David';<br>console.log(camper);<br>// logs 'David'</blockquote>",
+        "In a small application, you might not run into this type of problem, but when your code becomes larger, you might accidently overwrite a variable that you did not intend to overwrite. Because this behaviour does not throw an error, searching and fixing bugs becomes more difficult.",
+        "Another problem with the <code>var</code> keyword is that it is hoisted to the top of your code when it compiles. This means that you can use a variable before you declare it.",
+        "<blockquote>console.log(camper);<br>var camper = 'David';<br>// logs undefined</blockquote>",
+        "The code runs in the following order:",
+        "<ol><li>The variable <code>camper</code> is declared as undefined.</li><li>The value of <code>camper</code> is logged.</li><li>David is assigned to <code>camper</code>.</li></ol>",
+        "This code will run without an error.",
+        "A new keyword called <code>let</code> was introduced in ES6 to solve the problems with the <code>var</code> keyword. With the <code>let</code> keyword, all the examples we just saw will cause an error to appear. We can no longer overwrite variables or use a variable before we declare it. Some modern browsers require you to add <code>\"use strict\";</code> to the top of your code before you can use the new features of ES6.",
+        "Let's try using the <code>let</code> keyword.",
+        "<hr>",
+        "Fix the code so that it only uses the <code>let</code> keyword and makes the errors go away.",
+        "<strong>Note</strong><br>Remember to add \"use strict\" to the top of your code."
```

Comment:
You'll probably still want those escape characters `<code>\"use strict\";</code>`